docs/logs_tpm_module/log_with_tpm_pull_logs.txt

```diff
@@ -0,0 +1,6 @@
+RASPI_IP: pi@192.168.4.44
```

Comment:
this file isn't needed

Reply:
File deleted in commit `d552c2eaccbf5fc92778426c5f66aa93e54ace49`